modules/neo4j/src/test/resources/.gitignore

```diff
@@ -0,0 +1 @@
+container-license-acceptance.txt
```

Comment:
uh? Does it get created here? (e.g. by tests)

Reply:
I... do not know where this diff came from 😬 

Will remove!

Reply:
Done